models/users.js

```diff
@@ -83,6 +84,35 @@ const getJoinData = async (userId) => {
   }
 };
 
+const getSuggestedUsers = async (skill) => {
+  try {
+    const data = await itemModel.where("itemtype", "==", "USER").where("tagid", "==", skill).get();
+    const usersId = [];
+
+    if (!data.empty) {
+      data.forEach((doc) => {
+        usersId.push(doc.data().itemid);
+      });
+    }
+    const Promises = usersId.map((userId) => fetchUser({ userId }));
+    const resolved = await Promise.all(Promises);
+    const set = new Set();
+    const users = [];
+
+    resolved.forEach((resolve) => {
+      if (!set.has(resolve.user.id)) {
```

Comment:
can you please add a bit more descriptive names to the variable names of `Promises`, `resolved`, `set`, `resolve`